features/plugin.feature

```diff
@@ -1,25 +1,10 @@
 Feature: Test WordPress for plugins with known security issues
 
-  Scenario: A WordPress install with a plugin with a known security issue
-    Given a WP install
```

Comment:
Instead of removing the feature, could we just add another Given statement that checks for the presence of a WPScan key?

https://github.com/pantheon-systems/wp_launch_check/blob/main/features/steps/given.php

Reply:
The key is still present, and we don't plant to remove it right away.  When the key is present and rate limited, such is the the state of our token, the response from the API is that the plugin asked about is not vulnerable.

Reply:
I'm not sure I understand the rate limiting part, I'm just suggesting instead of removing the feature test, adding a new Given statement that checks for the presence of a key, and if the Given condition is not met, then that test just won't run. Is that accurate?

Reply:
I'm not really interested in maintaining a test for a scenario that is not very testable. The free tokens don't have enough capacity to allow for reliable tests. Hm, I wonder if Automattic might give us a special token that's unlimited for just one plugin.

Reply:
A free token can be used for up to 25 API requests a day, and this test would use 2 API calls it seems. Would we see running this more than 12 times a day?

https://wpscan.com/pricing

Reply:
In the work I've done on this plugin today, I've already exhausted the 25 free API calls for the free plugin. 
```
{
    "plan": "free",
    "requests_limit": 25,
    "requests_remaining": 0,
    "requests_reset": 1646956800
}
```

I'm ready to remove this test from the code.

Reply:
We could mock the service, but I think that's too much effort for the time being. Go ahead and remove the test.

Reply:
Yes, this sometimes runs more than 12 times a day; we were seeing intermittent failures, which is what brought us here.